dynamiq/callbacks/tracing.py

```diff
@@ -327,6 +327,8 @@ def on_flow_end(
             output_data, truncate_enabled=True
         )
         run.status = RunStatus.SUCCEEDED
+        if run.parent_run_id is None:
```

Comment:
Please leave a small comment here with definition that missed parent_run_id mean that run just single node